### PR TITLE
BEEM BEEM PEW PEW

### PR DESCRIPTION
**Pull request by chamoysvoice**
Hello Esteban-dido, 

Pls review este super feature :panda_face:

### CODE REVIEW

**Comment by esnilara** [comment]
nice!
:rainbow_flag: LGTM :shipit:🐑  🚢  🚀  📦

**Comment by esnilara** [comment]
LGTM
image

**Comment by benoror** [comment]
LGTM

giphy 4

File: panda-laser.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+BEEM BEEM
```
Comment (benoror) [inline panda-laser.txt]: `baz` missing

**Pull request closed without merging**